bundles/binding/org.openhab.binding.zwave/database/ge/zw3003.xml

```diff
@@ -87,9 +87,11 @@
 			<Maximum>255</Maximum>
 			<Size>1</Size>
 			<Label lang="en">On/Off Command Dim Rate</Label>
-			<Help lang="en">This value indicates in 10 millisecond resolution, how often the dim level will change. For example, if you set this parameter to 1, then every 10ms the dim level will change. If you set it to 255, then every 2.55 seconds the dim level will change.</Help>
+			<Help lang="en"><![CDATA[This value indicates in 10 millisecond resolution, <br>
```

Comment:
You shouldn't add ```<br>``` tags in places like this - just when you actually want to force an end-of-line. If you add breaks like this, then it means the browser/UI doesn't have the ability to format it as it sees fit - you've effectively hard formatted it.
So, only add the breaks where you actually want to enforce a new paragraph, or new point of some description.  Same comment further down in the file, but I won't repeat it...